src/repositories/changes.ts

```diff
@@ -18,14 +23,17 @@ export default function changeRepository(db: Db): ChangeRepository {
     },
     get: async (articleId: string, page = 0) => {
       const skip = page * MAX_PAGE_SIZE;
-      const changes = await db
+      const changesCursor = db
```

Comment:
Should this still not have `await`?

Reply:
No. Cursor is pretty much a builder pattern. You call a bunch of method. All chained to return a cursor. Only methods that return an actual result or count would not be chained and thus asynchronous